composer.json

```diff
@@ -38,8 +38,11 @@
   },
   "require-dev": {
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
+    "php-stubs/wp-cli-stubs": "^2.4",
     "phpunit/phpunit": "^6.5 || ^7.5",
-    "wearerequired/coding-standards": "^1.5"
+    "szepeviktor/phpstan-wordpress": "^0.6.5",
+    "wearerequired/coding-standards": "^1.5",
+    "wpackagist-plugin/glotpress": "^2.3.1"
```

Comment:
Can we please keep the plugin in the vendor directory instead of moving it into the default wp-content directory?

Reply:
@ocean90 Done.